### ISSUE TITLE
Copy, CopyOrCancel and Cut commands hang in Linux until you copy in some other app

### ISSUE THREAD

**Issue by fsackur** [PowerShell/PSReadLine#3884]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
N/A
```

### Screenshot

N/A

### Environment data

```console
PS Version: 7.4.0
PS HostName: ConsoleHost
PSReadLine Version: 2.3.4
PSReadLine EditMode: Emacs
OS: Linux ghetto-blaster 6.5.12-300.fc39.x86_64 #1 SMP PREEMPT_DYNAMIC Mon Nov 20 22:44:24 UTC 2023 x86_64 GNU/Linux
BufferWidth: 207
BufferHeight: 26
```

### Steps to reproduce

- Select some text
- Hit Ctrl-C or Ctrl-X

### Expected behavior

- Highlighting is cleared
- Control returns to the prompt

### Actual behavior

- The prompt hangs
- In other apps, the selected text can be copied
- When any other app copies to the clipboard, the hung PSReadLine shell returns control and the highlighting is cleared

**Comment by daxian-dbw** [comment]
@fsackur Sorry that this issue and #1993 slipped off my radar. I didn't realize the "copying to system clipboard" functionality had been always broken. I submitted the fix #3937, the artifact produced from CI can be found [here](https://ci.appveyor.com/project/SteveL-MSFT/psreadline/builds/49083933/artifacts). Please give it a try.

**Comment by fsackur** [comment]
I confirm the fix works on Fedora 39. Thanks @daxian-dbw for this, and for all the other work you do for us.
